src/QuizAtom.test.tsx

```diff
@@ -6,8 +6,8 @@ import { QuizAtom } from './QuizAtom';
 
 describe('QuizAtom', () => {
     it('should render', () => {
-        const { getByText } = render(<QuizAtom id="123abc" />);
+        const { getByText } = render(<QuizAtom id="123abc" questions={[]} />);
```

Comment:
The react testing library supports `fireEvent` for clicking, in case that's useful for this example! https://testing-library.com/docs/react-testing-library/cheatsheet

Probably a light test, as Cypress for an integration test in DCR is probably a nicer option for testing user behaviour